env.example

```diff
@@ -15,6 +18,8 @@ PORT=3000
 #   API_URL=https://dev.telescope.cdot.systems
 #   (our production server)
 #   API_URL=https://telescope.cdot.systems
+#   (AWS Cloud9 development)
+#   API_URL=${MY_PUBLIC_IP}:3000
```

Comment:
I'm having a problem with storing the public IP address as a variable in .env. 

If I hardcode the IP address like
`API_URL=http://35.174.16.133:3000`
the frontend will pick up the feeds from the backend with no problems but I don't want to hardcode it

The public IP address of the EC2 instance will change when it shuts down so I want it to have its own env variable.  Anyone know how I can do this?

Reply:
I'm not sure what the right approach to this is.